docs/contents/users/index.md

```diff
@@ -1,28 +1,20 @@
-## Create an EKS Distro Cluster
+## Using EKS Distro 
 
-The best way to create an EKS Distro cluster may be using one of our
-[Partners](../community/partners.md). There are links and instructions on that
-page for third party methods to install EKS-D.
+The best way to try out EKS Distro (EKS-D) is to use one of the
+[Partners](../community/partners.md) procedures to install and start an EKS-D cluster.
```

Comment:
Can you update the new location of the `partners.md` file here as well as in the `docs/contents/index.md`? Thanks!

Reply:
Good catch. Thanks. Done.

Reply:
Thanks! Just one more change - filename refers to `users/partners.md`, needs to be `users/install/partners.md` to align with the links.

Reply:
Okay. Fixed. Hopefully I got them all now. Let me know if I missed anything else.

Reply:
Thanks, the links all look good now. The `partners.md` is now at the users/ directory, it needs to be moved into the users/install directory. That should get the presubmit job passing.

Reply:
Done.